### PR TITLE
refactor: split ops.rs

### PR DESCRIPTION
**Pull request by bartlomieju**
Work towards #2730

This PR splits `//cli/ops.rs` into `//cli/ops/` module. I try to associate ops accordingly to their' counterparts in `//js/` (ref #2180)

CC @ry

### CODE REVIEW

File: cli/ops/mod.rs [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+  data.into()
+}
+
+#[inline]
+pub fn ok_future(buf: Buf) -> CliOpResult {
+  Ok(Op::Async(Box::new(futures::future::ok(buf))))
+}
+
+#[inline]
+pub fn ok_buf(buf: Buf) -> CliOpResult {
```
Comment (ry) [inline cli/ops/mod.rs]: I think this doesn't need to be pub ?

File: cli/ops/mod.rs [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+type CliDispatchFn = fn(
+  state: &ThreadSafeState,
+  base: &msg::Base<'_>,
+  data: Option<PinnedBuf>,
+) -> CliOpResult;
+
+pub type OpSelector = fn(inner_type: msg::Any) -> Option<CliDispatchFn>;
+
+#[inline]
+pub fn empty_buf() -> Buf {
```
Comment (ry) [inline cli/ops/mod.rs]: Maybe remove pub

**Comment by ry** [comment]
Looks good. I'm waiting on #2764 (and 0.14.1 release) before landing.

**Comment by bartlomieju** [comment]
> Looks good. I'm waiting on #2764 (and 0.14.1 release) before landing.

Awesome, couldn't help myself to split all of them. I'll rebase after the release :+1:

**Pull request merged**